IHP/IDE/CodeGen/ViewGenerator.hs

```diff
@@ -65,127 +66,157 @@ buildPlan' schema config =
                     |> head
                     |> fromMaybe []
 
+            -- when using the trimming quasiquoter we can't have another |] closure, like for the one we use with hsx.
+            qqClose = "|]"
+
+            viewHeader = [trimming|
+                module ${moduleName} where
+                import ${applicationName}.View.Prelude
+            |]
+                where
+                    moduleName = qualifiedViewModuleName config nameWithoutSuffix
+                    applicationName = get #applicationName config
+
+
+
+            genericView = [trimming|
+                ${viewHeader}
+                data ${nameWithSuffix} = {$nameWithSuffix}
+
+                instance View ${nameWithSuffix} where
+                    html ${nameWithSuffix} { .. } = [hsx|
+                        {breadcrumb}
+                        <h1>${nameWithSuffix}</h1>
+                        ${qqClose}
+                            where
+                                breadcrumb = renderBreadcrumb
+                                                [ breadcrumbWithLink "${pluralizedName}" ${indexAction}
+                                                , breadcrumbWithoutLink "${nameWithSuffix}"
+                                                ]
+            |]
+                where
+                    pluralizedName = pluralize name
+
+
+            showView = [trimming|
+                ${viewHeader}
+
+                data ShowView = ShowView { ${singularVariableName} :: ${singularName} }
+
+                instance View ShowView where
+                    html ShowView { .. } = [hsx|
+                        {breadcrumb}
+                        <h1>Show ${singularName}</h1>
+                        <p>{${singularVariableName}}</p>
+
+                    ${qqClose}
+                        where
+                            breadcrumb = renderBreadcrumb
+                                            [ breadcrumbWithLink "${pluralName}" ${indexAction}
+                                            , breadcrumbWithoutLink [hsx|Show ${singularName}${qqClose}
```

Comment:
To make this consistent we should use `""` here instead of `[hsx||]` on all calls :)

```suggestion
                                            , breadcrumbWithoutLink "Show ${singularName}"
```